### PR TITLE
Ignore disabled entries

### LINKED ISSUE #188: Don't show disabled world menu entries
As a user of Algernon, I don't want to see search results that don't do anything in order to speed up my workflow.

## Acceptance criteria
- [x] Don't show disabled world menu entries in result list

### PR DESCRIPTION
**Pull request by FranzSw**
closes #188

### CODE REVIEW

**Comment by bewee** [comment]
No idea why this happens. Tests are green on my machine. Probably just another CI problem? 😬

**Comment by FranzSw** [comment]
> No idea why this happens. Tests are green on my machine. Probably just another CI problem? 😬

Did you run it locally with smalltalk CI? Tests in our working images are not really reliable / reproducible

**Comment by TheGrayStone** [comment]
I think 'Enter Parent Project' is just not disabled in CI mode. Do you know why this menu entry is even there? Maybe we should rewrite the test to manually disable entries and then test that.

**Comment by bewee** [comment]
> I think 'Enter Parent Project' is just not disabled in CI mode. Do you know why this menu entry is even there? Maybe we should rewrite the test to manually disable entries and then test that.

Sounds good. We could reuse the already existing mock menu entry for that 🎉

**Comment by TheGrayStone** [comment]
> > I think 'Enter Parent Project' is just not disabled in CI mode. Do you know why this menu entry is even there? Maybe we should rewrite the test to manually disable entries and then test that.
> 
> Sounds good. We could reuse the already existing mock menu entry for that 🎉

Already on it!

File: packages/AlgernonTests.package/ALGTestTypes.class/instance/testWorldMenuDisabledAreExcluded.st [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+testWorldMenuDisabledAreExcluded
+	
+	| menu |
+	menu := ALGFakeDockingBarItemMorph addToDockingBar.
+	menu subMenu updateMenu.
+	self runQuery: 'Disabled dummy menu entry'.
+	self deny: (self resultsIncludeName: 'Disabled dummy menu entry').
+	menu abandon.
+	"self runQuery: 'Enter Parent Project'.
+	self deny: (self resultsIncludeName: 'Enter Parent Project')"
```
Comment (TheGrayStone) [inline packages/AlgernonTests.package/ALGTestTypes.class/instance/testWorldMenuDisabledAreExcluded.st]:
```suggestion
```

File: packages/Algernon.package/ALGNamedWorldMenuEntry.class/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+Instance Variables
+	morph:		<Object>
+	name:		<Object>
+
+morph
+	- xxxxx
+
+name
+	- xxxxx
```
Comment (TheGrayStone) [inline packages/Algernon.package/ALGNamedWorldMenuEntry.class/README.md]:
```suggestion
```

**Review by TheGrayStone — CHANGES_REQUESTED** [review]

File: packages/AlgernonTests.package/ALGTestTypes.class/instance/testWorldMenuDisabledAreExcluded.st (hunk 1 above)
Comment (bewee) [reply packages/AlgernonTests.package/ALGTestTypes.class/instance/testWorldMenuDisabledAreExcluded.st]: Trailing dot now

File: packages/AlgernonTests.package/ALGTestTypes.class/instance/testWorldMenuDisabledAreExcluded.st [hunk 3]
```diff
@@ -0,0 +1,9 @@
+testing world menu types
+testWorldMenuDisabledAreExcluded
+	
+	| menu |
+	menu := ALGFakeDockingBarItemMorph addToDockingBar.
+	menu subMenu updateMenu.
+	self runQuery: 'Disabled dummy menu entry'.
+	self deny: (self resultsIncludeName: 'Disabled dummy menu entry').
+	menu abandon.
```
Comment (FranzSw) [inline packages/AlgernonTests.package/ALGTestTypes.class/instance/testWorldMenuDisabledAreExcluded.st]:
```suggestion
	menu abandon
```

**Review by TheGrayStone — APPROVED** [review]

**Pull request merged**